scripts/resultsSum.pl

```diff
@@ -274,6 +275,10 @@ sub resultReporter {
 		$AQACertContent =~ s/\n/\n# /g;
 		print $fhOut "# AQACert.log content: \n# " . $AQACertContent . "\n";
 
+		#add machine name in TAP file
+		my $host = hostname;
+		print $fhOut "#Machine name: \n#" . $host . "\n";
```

Comment:
`print $fhOut "# Hostname: " . $host . "\n";	`  might be better. Update machine name to Hostname and remove the first `\n#`.

Reply:
Based on https://ci.adoptium.net/view/Test_grinder/job/Grinder/7059/artifact/aqa-tests/TKG/output_1680009017371/Grinder_20230328130836.tap/*view*/ it would also be good to add a 2nd `\n` at the end, to separate it from SHA.txt.

`print $fhOut "# Hostname: " . $host . "\n\n";`